api/google_api/util/SceGoogleApiHandler.js

```diff
@@ -61,35 +68,41 @@ class SceGoogleApiHandler {
    * This function prompts the user to visit the OAuth playground on Google
    * Cloud's website to supply an authorization code to it. After recieving the
    * code the function writes a JSON token the token file path;
+   * @param {boolean} isDevScript toggles this functions context: true for
+   * DevOps purposes, false for API endpoints.
    */
-  getNewToken() {
-    if (!this.runningInProduction) return;
+  getNewToken(isDevScript) {
```

Comment:
i would just generate the token every time, so remove this param

Reply:
Update: the purpose of the `isDevScript` parameter is because we want this script to run without having to have the server running. The script uses `getNewToken()`, which checks `if(!this.runningInProduction) return;`, so if we ran the script without the server running, it wouldn't work. The easiest fix is to add a boolean parameter and toggle it to true so that it would bypass that initial check and run successfully without needing the server running